### PR TITLE
Improve open source messaging and add Cody

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR adds mentions Cody being open-source with GH repo links to our extensions.

## Test plan

* Check links

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@cla-bot check

**Comment by MaedahBatool** [comment]
@cla-bot check

File: doc/getting-started/index.md [hunk 1]
```diff
@@ -57,10 +57,12 @@ You can also try [Sourcegraph.com](https://sourcegraph.com/search), which is a p
 
 Sourcegraph Enterprise is Sourcegraph’s primary offering and includes all code intelligence platform features. Sourcegraph Enterprise is the best solution for enterprises who want to use Sourcegraph with their organization’s code.
 
-Some third party extensions are OSS licensed (Apache 2), such as:
-1. Sourcegraph browser extension
-2. Sourcegraph VS Code extension
-3. Sourcegraph Jetbrains extension
+[Cody](https://docs.sourcegraph.com/cody), Sourcegraph's AI-enabled editor assistant, is free and [open-source](https://about.sourcegraph.com/blog/open-sourcing-cody) and is licensed under the Apache 2.0 license.
```
Comment (jdorfman) [inline doc/getting-started/index.md:60]: we use "open source" in our style guide which alerts me with Grammarly


Image 2023-07-05 at 10 31 42 AM png

**Comment by jdorfman** [comment]
@MaedahBatool need to add testing plan to the first comment. You can say:

```
## Test plan

* Check links

<!-- All pull requests REQUIRE a test plan: https://docs.sourcegraph.com/dev/background-information/testing_principles -->
```

File: doc/getting-started/index.md (hunk 1 above)
Comment (MaedahBatool) [reply doc/getting-started/index.md]: Done 👍

**Comment by MaedahBatool** [comment]
> @MaedahBatool need to add testing plan to the first comment. You can say:
> 
> ```
> ## Test plan
> 
> * Check links
> 
> 
> ```

Oh, thanks for the help. Added ✅

**Review by jdorfman — APPROVED** [review]

**Pull request merged**